dipy/data/fetcher.py

```diff
@@ -610,20 +616,42 @@ def read_mni_template(contrast="T2"):
 
     Examples
     --------
-    Get only the T2 file:
-    >>> T2_nifti = read_mni_template("T2") # doctest: +SKIP
-    Get both files in this order:
-    >>> T1_nifti, T2_nifti = read_mni_template(["T1", "T2"]) # doctest: +SKIP
+    Get only the T1 file for version c:
+    >>> T1_nifti = read_mni_template("c", contrast = "T1") # doctest: +SKIP
+    Get both files in this order for version a:
+    >>> T1_nifti, T2_nifti = read_mni_template("a", contrast = ["T1", "T2"]) # doctest: +SKIP
     """
     files, folder = fetch_mni_template()
-    file_dict = {"T1": pjoin(folder, 'mni_icbm152_t1_tal_nlin_asym_09a.nii'),
-                 "T2": pjoin(folder, 'mni_icbm152_t2_tal_nlin_asym_09a.nii')}
-    if isinstance(contrast, str):
-        return nib.load(file_dict[contrast])
+    file_dict_a = {"T1": pjoin(folder, 'mni_icbm152_t1_tal_nlin_asym_09a.nii'),
+                   "T2": pjoin(folder, 'mni_icbm152_t2_tal_nlin_asym_09a.nii')}
+
+    file_dict_c = {
+        "T1": pjoin(
+            folder, 'mni_icbm152_t1_tal_nlin_asym_09c.nii'), "mask": pjoin(
+            folder, 'mni_icbm152_t1_tal_nlin_asym_09c_mask.nii')}
+
+    if contrast == "T2" and version == "c":
```

Comment:
You are not covering however the case:
T1_nifti, T2_nifti = read_mni_template("c", contrast = ["T1", "T2"])